pkg/controller/plan/kubevirt.go

```diff
@@ -1265,6 +1276,14 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 							Protocol:      core.ProtocolTCP,
 						},
 					},
+					SecurityContext: &core.SecurityContext{
+						AllowPrivilegeEscalation: &allowPrivilageEscalation,
+						RunAsNonRoot:             &nonRoot,
```

Comment:
This can be moved to `Pod.SecurityContext` to avoid duplication.

Reply:
It worked for me, please re-check. The documentation states that:

>  May also be set in SecurityContext.  If set in both SecurityContext and PodSecurityContext, the value specified in SecurityContext takes precedence.

Reply:
Yes, I am not saying we don't. But we already have `PodSecurityContext` with some configuration so it makes sense to set the fields that can be set there instead of duplicating that per container.